packages/frontend/src/styles/index.css

```diff
@@ -1,3 +1,16 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+html,
+body {
```

Comment:
Not sure how comfortable I am having every page being this gradient background, it's quite difficult to design for.

I think these starter pages are fine, but the main app should be white background (if light mode)

Reply:
You should be able to create a div on your page that takes up the full page `h-screen` and set the colour on that div.